README.md

```diff
@@ -263,6 +263,18 @@ export default Ember.Route.extend({
 
 If your errors aren't standard, the helper function for that error type can be used as the base to build your custom detection function.
 
+#### Access the response in case of error
+
+If you need to access the json response of a request that failed, you can use the `raw` method instead of `request`.
+
+```js
+this.get('ajax').raw(url, options)
+  .then(response => this.handleSuccess(response))
```

Comment:
I thought the `.catch` part was the only one receiving all the extra info aside from the response. But if `.then` also receives the same as `.catch`, then yes, you're right. I will confirm from the codebase, but if you can positively confirm before, let me know.